sickbeard/helpers.py

```diff
@@ -1429,10 +1428,8 @@ def getURL(url, post_data=None, params=None, headers=None,  # pylint:disable=too
 
     try:
         resp.raise_for_status()
-    except requests.RequestException as e:
+    except requests.exceptions.RequestException as e:
```

Comment:
@medariox shouldn't we add both? `requests.RequestException` and `requests.exceptions` ?